ts/src/test/Exchange/base/test.sharedMethods.ts

```diff
@@ -74,7 +74,7 @@ function assertStructure (exchange, skippedProperties, method, entry, format, em
             if (key in skippedProperties) {
                 continue;
             }
-            assert (key in entry, 'key "' + stringValue (key) + '" is missing from structure' + logText);
+            assert (key in entry, '"' + stringValue (key) + '" key is missing from structure' + logText);
```

Comment:
We may need an extra space before logText.

Reply:
> We may need an extra space before logText.

the thing is that `logText` is already formed with needed separator & space in front, so we just easily concatenate with other strings without adding extra separators on each line.

Reply:
even though build didnt pass, this PR is not cause, and you can consider this as ready.